charts/hdfs-datanode-k8s/README.md

```diff
@@ -9,13 +9,25 @@ HDFS `datanodes` running inside a kubernetes cluster. See the other chart for
   $ kubectl label node YOUR-MASTER-NAME hdfs-datanode-exclude=yes
   ```
 
-  2. Launch this helm chart, `hdfs-datanode-k8s`.
+  2. (Skip this if you do not plan to enable Kerberos)
+     Conduct the Kerberos setups described in the namenode README.md, if you
```

Comment:
It's better to add a link for README.md here.

Reply:
Good idea. Thanks.